test/01_WETH10.test.js

```diff
@@ -111,6 +111,10 @@ contract('WETH10', (accounts) => {
         events[0].returnValues.data.should.equal('0x11')
       });
 
+      it('should not transfer to the contract address', async () => {
```

Comment:
Same test for `sendTransaction`, `deposit`, `depositTo`, etc. Easy to do copy & paste and test all those `require` in one go.

Reply:
Not possible for sendTransaction/deposit. I've added tests for all functions that could potentially send assets (ETH or WETH) to the contract